main/global_state.h

```diff
@@ -60,7 +62,7 @@ typedef struct
     char best_session_diff_string[DIFF_STRING_SIZE];
     bool FOUND_BLOCK;
     bool startup_done;
-    char ssid[20];
```

Comment:
hooo shit, did we had this bad bug !!!! I hate C letting you down on this simple yet very common problem.

good spotted there